dev_guide/projects.adoc

```diff
@@ -194,3 +194,98 @@ To delete a project using the CLI:
 ----
 $ oc delete project <project_name>
 ----
+
+ifdef::openshift-online[]
+[[project-collaboration-in-online-pro]]
+== Project Collaboration in {product-title} Pro
+
+Every {product-title} Pro account has the ability to add up to 50 collaborator
+users per subscription. These collaborator users are granted cluster access from
+{product-title} Pro account subscribers so that they can collaborate on projects
+hosted with {product-title}. This allows multiple users to gain access to
+projects under a single subscription, without having to pay a monthly fee for
+every account.
+
+[NOTE]
+====
+Existing {product-title} Pro users do not need to be added as collaborators.
+====
+
+Collaboration is not available in {product-title} Starter.
+
+[[collaboration-restrictions]]
+=== Collaboration Restrictions
+
+Collaborators only have access to the specific projects (on clusters owned by
+the {product-title} Pro subscriber) that they are granted access to. They can
+not add projects of their own ownership. Collaborators can only add resources.
```

Comment:
> Collaborators can only add resources.

This is not entirely correct. We should say something on the lines of:
Collaborators can only access the resources within the projects that they have been granted access to. Also, their ability to view/edit/manage the project resources will depend on the specific role that they have been granted within the project.